src/components/axes/timeAxis.ts

```diff
@@ -445,6 +480,20 @@ export module Axis {
       return this;
     }
 
+    private _hideOverflowingTiers() {
+      var availableHeight = this.height();
+      var usedHeight = 0;
+
+      var visibleAxisTiers = this._element
+        .selectAll(".time-axis-tier")
+        .filter(function(d: Element, i: number) {
+          return d3.select(this).style("visibility") === "visible";
```

Comment:
You use `"visible"` here, but `"inherit"` elsewhere. We probably should use one of them everywhere (probably `"inherit"`).